lib/iex/lib/iex/introspection.ex

```diff
@@ -115,13 +126,37 @@ defmodule IEx.Introspection do
         end
         :ok
       else
-        :not_found
+        if has_callback?(mod, fun, arity) do
+          :behaviour_found
+        else
+          :not_found
+        end
```

Comment:
I think it's fine to do this on a different PR since this PR is in line with the rest of the file IMO. :)